File: programacion-1/trabajo-practico-2/Ejercicio3.java
public class Ejercicio3 {
/*
Escribir un programa que solicite el ingreso de un número mayor a
50, y lo muestre por pantalla en caso de ser múltiplo de 2 o 3, sino
muestre un mensaje que informe dicha situación.
*/
    public static void main(String[] args) {
        
        int numero = 0;

        System.out.println("Ingrese un numero mayor a 50");
        numero = Utils.leerInt();

        if((numero > 50) && ((numero % 2 == 0) || (numero % 3 ==0))){
            System.out.println("su numero ingresado es: " + numero);
        }
        else{
            System.out.println("error");
        }
    }
}

File: taller-matematica-computacional/trabajo-practico-1/Ejercicio3.py
'''
3. Escribir una función en Python que determine si un número dado es múltiplo de 2.
'''

MULTIPLO: int = 2

numero: int = 5

if numero % MULTIPLO == 0:
    print(f"El numero {numero} es multiplo de {MULTIPLO}")
else:
    print("El numero {numero} no es multiplo de {MULTIPLO}")
    